aeppl/transforms.py

```diff
@@ -285,67 +365,82 @@ def log_jac_det(self, value, *inputs):
 # ======================
 
 
-TransformedUniformRV = create_default_transformed_rv_op(
+TransformedUniformRV = create_transformed_rv_op(
```

Comment:
These renamings appear to be undoing renamings from an earlier commit of yours: 52dd503.  Is this intentional, or are these changes unintended regressions that snuck into the commit?

Reply:
Intentional, the previous change made the name better reflect was the method was doing at the time of the previous commit, and the same applies here.

Reply:
They are not an exact revert. It has the new `op` suffix